src/routes/employees.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import Employees from 'Components/Employees/index';
+import EmployeesForm from 'Components/Employees/Form';
+import EmployeesHome from 'Components/Employees/Home';
+import EmployeeTimeSheets from 'Components/Employees/TimeSheets';
+import EmployeeProfile from 'Components/Employees/Profile';
+
+const Employee = () => {
+  return (
+    <Switch>
+      <Route exact path="/employees" component={Employees} />
+      <Route path="/employees/home" component={EmployeesHome} />
+      <Route path="/employees/timesheets" component={EmployeeTimeSheets} />
+      <Route path="/employees/profile" component={EmployeeProfile} />
```

Comment:
add this line in between

```suggestion
      <Route path="/employees/timesheets/:id" component={EmployeeProfile} />
```